tests/turbine_unit_test.py

```diff
@@ -354,61 +372,61 @@ def test_axial_induction():
     turbine = Turbine.from_dict(turbine_data)
     turbine_floating = Turbine.from_dict(turbine_floating_data)
     turbine_type_map = np.array(N_TURBINES * [turbine.turbine_type])
-    turbine_type_map = turbine_type_map[None, None, :]
+    turbine_type_map = turbine_type_map[None, :]
 
     baseline_ai = 0.25116283939089806
 
     # Single turbine
     wind_speed = 10.0
     ai = axial_induction(
-        velocities=wind_speed * np.ones((1, 1, 1, 3, 3)),
-        yaw_angle=np.zeros((1, 1, 1)),
-        tilt_angle=np.ones((1, 1, 1)) * 5.0,
-        ref_tilt_cp_ct=np.ones((1, 1, 1)) * 5.0,
+        velocities=wind_speed * np.ones((1, 1, 3, 3)), # 1 findex, 1 Turbine
+        yaw_angle=np.zeros((1, 1)),
+        tilt_angle=np.ones((1, 1)) * 5.0,
+        ref_tilt_cp_ct=np.ones((1, 1)) * 5.0,
         fCt={turbine.turbine_type: turbine.fCt_interp},
         tilt_interp=np.array([(turbine.turbine_type, None)]),
-        correct_cp_ct_for_tilt=np.array([[[False]]]),
-        turbine_type_map=turbine_type_map[0,0,0],
+        correct_cp_ct_for_tilt=np.array([[False]]),
+        turbine_type_map=turbine_type_map[0,0],
     )
     np.testing.assert_allclose(ai, baseline_ai)
 
     # Multiple turbines with ix filter
     ai = axial_induction(
-        velocities=np.ones((N_TURBINES, 3, 3)) * WIND_CONDITION_BROADCAST,  # 3 x 4 x 4 x 3 x 3
-        yaw_angle=np.zeros((1, 1, N_TURBINES)),
-        tilt_angle=np.ones((1, 1, N_TURBINES)) * 5.0,
-        ref_tilt_cp_ct=np.ones((1, 1, N_TURBINES)) * 5.0,
+        velocities=np.ones((N_TURBINES, 3, 3)) * WIND_CONDITION_BROADCAST,  # 12 x 4 x 3 x 3
```

Comment:
`print(((N_TURBINES, 3, 3) * WIND_CONDITION_BROADCAST).shape)` -> (12, 1, 1, 3)

Reply:
Are you sure?  I just ran in debug mode and get:

np.shape(np.ones((N_TURBINES, 3, 3)) * WIND_CONDITION_BROADCAST)
(12, 4, 3, 3)

Reply:
Yep you're right, I must have had some changes in my side that I didn't account for. Thanks for verifying.